### ISSUE TITLE
Shoot an image and tranfer image from camera to pc

### ISSUE THREAD

**Issue by thanos91** [genicam/harvesters#108]
**Is your feature request related to a problem? Please describe.**
 I need to create a GUI application that with the press of a button will shoot the camera and obtain the image. For someone that does not know the GenICam protocol, it is really hard to follow the instructions on this module. Do your instructions handle the aforementioned case?

**Describe the solution you'd like**
A clear example that would handle
a) Connection with the camera and shooting an image
b) Manage a transfer of the image from the camera to the hard disk in a computer

Minimal working example, in opencv case would be

```
import cv2
import matplotlib.pyplot as plt

cap = cv2.VideoCapture(0)

#Check whether user selected camera is opened successfully.
if cap.isOpened():
    time.sleep(2)

    ret, frame = cap.read()

    # Display the resulting frame
    plt.imshow(frame)




```

**Comment by kazunarikudo** [comment]
Hi. Maybe you should try to walk through the following example first:

https://github.com/genicam/harvesters#id54

And then, try to digest each line does reading through the following section:

https://github.com/genicam/harvesters#id56

Finally, if you are ready to implement your GUI, the following project may help you as a reference:

https://github.com/genicam/harvesters_gui

Good luck.

/Kazunari
